sys/include/ztimer.h

```diff
@@ -329,7 +329,10 @@ struct ztimer_clock {
     ztimer_now_t checkpoint;        /**< cumulated time at last now() call  */
 #endif
 #if MODULE_PM_LAYERED || DOXYGEN
+    union{
     uint8_t required_pm_mode;       /**< min. pm mode required for the clock to run */
+    uint8_t block_pm_mode;          /**< min. pm mode to block for the clock to run */
+    };
```

Comment:
i am not sure how internal and not internal API is decided but core/lifo i going through a deprecation cycle for maybe beeing not internal even though no one could name a valid usecase

Reply:
I personaly think this is only used via the PM_MODE defines and therefor internal but since the require does not take up any space i am ok with deprecation and deletion

Reply:
Deprecation comes at a cost: This increases complexity and can cause confusion. Also, new users of deprecated API occasionally still pop up even many months after. So, if there is no (valid) external use, we should IMO avoid the effort.

Maybe @kaspar030 and @bergzand have an opinion on this?